src/html/components/forms.html

```diff
@@ -5,31 +5,66 @@ <h2 id="style-guide-form" class="style-guide-title">Form Fields</h2>
 	<label for="form-field-text">Text Field</label>
 	<input id="form-field-text" type="text" />
 
-	<label for="form-field-search">Standard Search Field</label>
-	<input id="form-field-search" type="search" />
+	<label for="form-field-text-large">Large Code Field</label>
+	<input id="form-field-text-large" type="text" class="Form_Field Form_Field-Lg Form_Field-Code" placeholder="Placeholder" />
 
-	<label for="form-field-email">Email Form Field</label>
-	<input id="form-field-email" type="email" />
+	<label for="form-field-text-md">Medium Password Field</label>
+	<input id="form-field-text-md" type="password" class="Form_Field Form_Field-Md" placeholder="Placeholder"/>
 
-	<label for="form-field-password">Password Field</label>
-	<input id="form-field-password" type="password" />
+	<label for="form-field-text-sm">Small Search Field</label>
+	<input id="form-field-text-sm" type="search" class="Form_Field Form_Field-Sm" placeholder="Plchldr"/>
 
-	<label for="form-field-code">Large Code Field</label>
-	<input id="form-field-code" class="Form_Field-Code Form_Field-Lg" type="text" />
+	<label for="form-field-textarea">Textarea</label>
+	<textarea id="form-field-textarea"></textarea>
 
-	<label for="form-field-text-large">Large Form Field</label>
-	<input id="form-field-text-large" type="text" class="Form_Field Form_Field-Lg" placeholder="Placeholder" />
+	<fieldset>
 
-	<label for="form-field-text-md">Medium Form Field</label>
-	<input id="form-field-text-md" type="text" class="Form_Field Form_Field-Md" placeholder="Placeholder"/>
+		<legend>Radio</legend>
 
-	<label for="form-field-text-sm">Small Form Field</label>
-	<input id="form-field-text-sm" type="text" class="Form_Field Form_Field-Sm" placeholder="Plchldr"/>
+		<div class="Form_Radio_Group">
+			<input id="form-field-radio-1" type="radio" name="choice" class="Radio" value="choice 1" checked>
+			<label for="form-field-radio-1">Choice 1</label>
+		</div>
 
-	<label for="form-field-textarea">Textarea</label>
-	<textarea id="form-field-textarea"></textarea>
+		<div class="Form_Radio_Group">
+			<input id="form-field-radio-2" type="radio" name="choice" class="Radio" value="choice 2" checked>
+			<label for="form-field-radio-2">Choice 2</label>
+		</div>
+
+		<div class="Form_Radio_Group">
+			<input id="form-field-radio-3" type="radio" name="choice" class="Radio" value="choice 3" checked>
+			<label for="form-field-radio-3">Choice 3</label>
+		</div>
+
+	</fieldset>
+
+	<fieldset>
+
+		<legend>Checkbox</legend>
+
+		<div class="Form_Checkbox_Group">
+			<input id="checkbox_1" type="checkbox" value="first_checkbox" name="item_1" class="Checkbox" checked value="1">
```

Comment:
Updated it. Values don't actually matter because its not a real form - but fixed them so we set a good example!

Prechecked - thats a good idea.